airbyte-integrations/connectors/source-lever-hiring/integration_tests/spec.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
also please annotate oauth parameters like described in this issue #6166

Reply:
Done